code/game/turfs/simulated/wall_types.dm

```diff
@@ -66,7 +66,13 @@
 /turf/simulated/wall/plastic/facade
 	girder_material = /decl/material/solid/organic/plastic
 
-/turf/simulated/wall/sandstone
+/turf/simulated/wall/brick
```

Comment:
should this be an abstract type? if not, it should have `icon_state = "stone"` and a material set.